learning/tour-of-beam/frontend/README.md

```diff
@@ -17,47 +17,62 @@
      under the License.
  -->
 
- # Tour of Beam
- These are the main sources of the Tour of Beam website.
+# Tour of Beam
 
- # About
+These are the main sources of the Tour of Beam website.
+
+# About
 
 ## Getting started
+
 Running, debugging, and testing all require this first step that fetches
 dependencies and generates code:
 
 ```bash
 cd ../../../playground/frontend/playground_components
 flutter pub get
-flutter pub run build_runner build
 cd ../../../learning/tour-of-beam/frontend
 flutter pub get
-flutter pub run build_runner build
```

Comment:
Revert? Looks like it is needed.

Reply:
@alexeyinkin and I have updated README.

Reply:
Based on the [instructions](https://github.com/akvelon/beam/blob/dc4e142f25e1c7efa23b26a04158e1f208ea3864/learning/tour-of-beam/frontend/README.md?plain=1#L37-L39), received the following error:

```
No .dart_tool/package_config.json file found, please run "flutter pub get".

Starting with Dart 2.7, the package_config.json file configures resolution of package import URIs; run "flutter pub get" to generate it.
pub finished with exit code 65
```

So ran, `flutter pub get` .  FYI I saw a new file to be staged in git:

```
git status
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   pubspec.lock
```

Then followed the original instructions: `flutter pub run build_runner build` and received the error:

```
[SEVERE] Conflicting outputs were detected and the build is unable to prompt for permission to remove them. These outputs must be removed manually or the build can be run with `--delete-conflicting-outputs`. The outputs are: lib/models/unit_content.g.dart
lib/models/unit_progress.g.dart
lib/repositories/models/get_sdks_response.g.dart
lib/repositories/models/group.g.dart
lib/repositories/models/get_user_progress_response.g.dart
lib/repositories/models/node.g.dart
lib/repositories/models/module.g.dart
lib/repositories/models/unit.g.dart
lib/repositories/models/get_content_tree_response.g.dart
lib/auth/method.g.dart
pub finished with exit code 78
```

Steps to reproduce:

1) git clone -b issue23692_auth https://github.com/akvelon/beam.git issue23692_auth
2) Follow https://github.com/akvelon/beam/blob/dc4e142f25e1c7efa23b26a04158e1f208ea3864/learning/tour-of-beam/frontend/README.md?plain=1#L37-L39

Reply:
I've added `flutter pub run build_runner build --delete-conflicting-outputs`.

Reply:
I've committed a fresh `pubspec.lock` file.

Reply:
@damondouglas this README is anyway temporary because Gradle tasks are also in the line for the review:
- https://github.com/apache/beam/pull/24919